package.json

```
@@ -2,7 +2,7 @@
   "name": "axios",
   "version": "1.6.8",
   "description": "Promise based HTTP client for the browser and node.js",
-  "main": "index.js",
+  "main": "dist/node/axios.cjs",
```

I think instead of replacing you may want to keep them both, supporting mjs entry and the cjs. although the actual top level main for axios is `dist/axios.js`

  "main": "dist/node/axios.cjs",
```suggestion
  "main": "dist/axios.js",
  "module": "index.js",
```

Does that make sense to you as well?